npg_qc_viewer/lib/npg_qc_viewer/Controller/QcOutcomes.pm

```diff
@@ -158,12 +158,42 @@ sub _update_outcomes {
 
     try {
       my $seq_outcomes = $data->{$SEQ_OUTCOMES} || {};
-      my $lane_info = keys %{$seq_outcomes} ?
-        $self->_lane_info($c->model('MLWarehouseDB'), $seq_outcomes) : {};
+      my @lane_rps = keys %{$seq_outcomes};
+      my $lane_info = {};
+      # Why do we have to know the manufacturer?
+      # We are pushing Element Biosciences pooled libraries through this system.
+      # For time being (May 2025) no library-level autoqc data is available.
+      # The lanes' QC outcomes has to be finalised in the absenve of plex-level
+      # QC outcomes. 
+      my $illumina_instr_manufacturer = 1;
+      if (@lane_rps) {
+         my @inflated_lane_rps =
+           map { npg_tracking::glossary::rpt->inflate_rpt($_) }
+           @lane_rps;
+         my @id_runs = uniq map { $_->{'id_run'} } @inflated_lane_rps;
```

Comment:
I'm tempted to collapse these into one line, since `@inflated_lane_rps` is not used again.